Sources/UAPusher/Builder/Campaigns.swift

```diff
@@ -0,0 +1,51 @@
+
+import JSON
+
+/// Campaigns segment of an Urban Airship push notification payload
+/// See: https://docs.urbanairship.com/api/ua/#campaigns-object
+public final class Campaigns: BuildingBlock {
```

Comment:
What's your thoughts behind the names (singular vs plural) of these building blocks?

Reply:
I named them after the naming on Urban Airship. See https://docs.urbanairship.com/api/ua/#push-object